Failing Tensorflow test TestTensorFlow_GetImageTags

When I execute `go test -tags slow -v ./internal/photoprism/...`, test `TestTensorFlow_GetImageTags` fails with the following message:

```
=== RUN   TestTensorFlow_GetImageTags
--- FAIL: TestTensorFlow_GetImageTags (2.59s)
    config.go:77: Initializing test data
    tensorflow_slow_test.go:33:
        	Error Trace:	tensorflow_slow_test.go:33
        	Error:      	Not equal:
        	            	expected: 0.1648176
        	            	actual  : 0.16481775
        	Test:       	TestTensorFlow_GetImageTags
```

I assume this is due to the recent tensorflow update. Project seems to be running fine on travis.

--- a/internal/photoprism/tensorflow_slow_test.go
+++ b/internal/photoprism/tensorflow_slow_test.go
@@ -4,6 +4,7 @@ package photoprism
 
 import (
 	"io/ioutil"
+	"math"
 	"testing"
 
 	"github.com/photoprism/photoprism/internal/test"
@@ -30,6 +31,6 @@ func TestTensorFlow_GetImageTags(t *testing.T) {
 		assert.Equal(t, "tabby", result[0].Label)
 		assert.Equal(t, "tiger cat", result[1].Label)
 
-		assert.Equal(t, float32(0.16481775), result[1].Probability)
+		assert.Equal(t, float64(0.165), math.Round(float64(result[1].Probability)*1000)/1000)
 	}
 }

--- a/internal/photoprism/tensorflow_slow_test.go
+++ b/internal/photoprism/tensorflow_slow_test.go
@@ -30,6 +30,6 @@ func TestTensorFlow_GetImageTags(t *testing.T) {
 		assert.Equal(t, "tabby", result[0].Label)
 		assert.Equal(t, "tiger cat", result[1].Label)
 
-		assert.Equal(t, float32(0.1648176), result[1].Probability)
+		assert.Equal(t, float32(0.16481775), result[1].Probability)
 	}
 }

--- a/internal/photoprism/tensorflow_test.go
+++ b/internal/photoprism/tensorflow_test.go
@@ -1,6 +1,7 @@
 package photoprism
 
 import (
+	"math"
 	"testing"
 
 	"github.com/photoprism/photoprism/internal/test"
@@ -24,5 +25,5 @@ func TestTensorFlow_GetImageTagsFromFile(t *testing.T) {
 	assert.Equal(t, "tabby", result[0].Label)
 	assert.Equal(t, "tiger cat", result[1].Label)
 
-	assert.Equal(t, float32(0.16481775), result[1].Probability)
+	assert.Equal(t, float64(0.165), math.Round(float64(result[1].Probability)*1000)/1000)
 }

--- a/internal/photoprism/tensorflow_test.go
+++ b/internal/photoprism/tensorflow_test.go
@@ -24,5 +24,5 @@ func TestTensorFlow_GetImageTagsFromFile(t *testing.T) {
 	assert.Equal(t, "tabby", result[0].Label)
 	assert.Equal(t, "tiger cat", result[1].Label)
 
-	assert.Equal(t, float32(0.1648176), result[1].Probability)
+	assert.Equal(t, float32(0.16481775), result[1].Probability)
 }

